spec/menu_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'menu'
+
+describe Menu do
+
+  let(:takeaway) { double :takeaway }
+  let(:menu) { double(:menu, show: dishes_list) }
```

Comment:
Oops - you're supposed to be testing the `Menu` class.  Now you're just testing a double.